google/cloud/pubsub_v1/subscriber/_protocol/dispatcher.py

```diff
@@ -66,6 +67,13 @@
 IDs at a time.
 """
 
+_MIN_ACK_MODACK_RETRY_DURATION_SECS = 1
+"""The time to wait for the first retry of failed acks and modacks when exactly-once
```

Comment:
Should the variable name contain "exactly_once"

Reply:
Changed.

Reply:
Grep'd and updated uses of "exactly-once" to include the "delivery" part.